mutations/core.py

```diff
@@ -102,6 +116,17 @@ def _get_input(self, field):
             return self.fields[field].default
         return
 
+    def _execute_async(self):
+        try:
+            loop = asyncio.get_running_loop()
+        except RuntimeError:
+            loop = None
+
+        if loop and loop.is_running():
+            return RunThread(self.execute).result
+        else:
+            return asyncio.run(self.execute())
+
```

Comment:
need to check if any event loops are already running  - https://stackoverflow.com/questions/55409641/asyncio-run-cannot-be-called-from-a-running-event-loop-when-using-jupyter-no